interview-questions.md

```diff
@@ -7,60 +7,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: git is a software native to the apple OS which allows version control for files locally on the computer. Github is an online resource which interacts with git to allow multiple users to simultaneously work on and update files, under the protection of version control. 
 
+  Researched answer: git is an open source software (available for several different OS's) which allows for version control of projects of varying sizes and scopes. Github is a free code hosting platform for version control and collaboration.
 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: The phrasing of this question kind of confused to me. I'm interpreting this as asking for which operators can be used to give Boolean values. The following operators will yield Boolean values: <, <=, >, >=, !=, !==, ==, ===, ||, and &&. In addition, the || and && operators can be chained together for evaluating multiple statements, which will result in a single Boolean value.
```

Comment:
If you have questions, reaching out to your cohort slack channel is always a good move.
I am looking for relational, logical, and equality operators